cvmfs/sync_item.h

```diff
@@ -57,13 +58,28 @@ class SyncItem {
   inline bool WasSymlink()        const { return WasType(kItemSymlink);        }
   inline bool IsNew()             const { return WasType(kItemNew);            }
   inline bool IsCharacterDevice() const { return IsType(kItemCharacterDevice); }
+  inline bool IsBlockDevice()     const { return IsType(kItemBlockDevice);     }
   inline bool IsGraftMarker()     const { return IsType(kItemMarker);          }
   inline bool IsExternalData()    const { return external_data_;               }
 
   inline bool IsWhiteout()        const { return whiteout_;                    }
   inline bool IsCatalogMarker()   const { return filename_ == ".cvmfscatalog"; }
   inline bool IsOpaqueDirectory() const { return IsDirectory() && opaque_;     }
 
+  inline bool IsSpecialFile()     const {
+    return IsCharacterDevice() || IsBlockDevice();
+  }
+
+  inline unsigned int GetMajor()     const {
```

Comment:
I think it would be better to call it `GetRdevMajor` and `GetRdevMinor` to avoid confusion which of the devices in the stat structure is meant.